### PR TITLE
Fixed all bunk name:* properties

### LINKED ISSUE #912: A few wrong name labels
1. Locality 85688069 has  a typo in one of the name labels: rus_x_**prefered**.
2. Locality 1108782561 has  a wrong name types: eng_x_historic. I think it should be eng_x_**historical** instead.
3. Macrohood 102147495 (BoCaCa) has  a name label in the old format as well as one in the current format. I think the old "name:eng_p" label should be removed.

With thanks,
Laszlo

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #912.

This PR updates bad `name:*` properties. These properties include:

 - `name:rus_x_prefered`
 - `name:eng_x_historic`
 - `name:eng_p`

No PIP work required; straight property updates only.

Script/logic used to remap the properties, here: https://github.com/mapzen/whosonfirst-toolbox/commit/567570fd4893ff807efcaefb60107a17fc5b81ec

### CODE REVIEW

File: data/858/929/15/85892915.geojson [hunk 1]
```diff
@@ -50,6 +50,9 @@
     "name:eng_x_preferred":[
         "BoCoCa"
     ],
+    "name:eng_x_variant":[
+        "BoCoCa"
```
Comment (nvkelso) [inline data/858/929/15/85892915.geojson]: Please don't duplicate preferred value into variant list.

**Comment by stepps00** [comment]
Looks like the `name:eng_p` issue in `85892915.geojson` was recently taken care of here: https://github.com/whosonfirst-data/whosonfirst-data/commit/877f102a9cb417ffd032dbccca454fd446bd078b

The most recent commit removed the duplicate value in the `name:eng_x_variant` property for BoCoCa. This should be good to merge now.

**Comment by thisisaaronland** [comment]
I fixed the `name:eng_p` stuff as a side-effect of writing this:

https://github.com/whosonfirst/go-whosonfirst-names-tools#wof-names-validate

**Comment by stepps00** [comment]
https://whosonfirst.mapzen.com/boundaryissues/pipeline/330

**Pull request merged**